Lib/test/test_ctypes/test_numbers.py

```diff
@@ -7,6 +8,10 @@
                     c_char, c_byte, c_ubyte, c_short, c_ushort, c_int, c_uint,
                     c_long, c_ulong, c_longlong, c_ulonglong,
                     c_float, c_double, c_longdouble, c_bool)
+try:
+    from ctypes import c_double_complex
```

Comment:
You can move this import inside test_complex, or just write ctypes.c_double_complex in the test.